### ISSUE TITLE
MIMIC-IV Notes: Censoring of echo observations as PHI

### ISSUE THREAD

**Issue by alexander-lacki** [MIT-LCP/mimic-code#1507]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    [X] Checked the online documentation: https://mimic.mit.edu/
    [X] Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description

Hi,
thank you very much for providing and maintaining this database!

I would like to point to an issue that appears to be present only in MIMIC-IV, that was not a problem in MIMIC-III. 
In our project we are operating on the free-text clinical notes. In MIMIC-III these used to include information regarding atrial dilation, which could easily be string matched. The notes were written in the form:
> LEFT ATRIUM: Mild LA enlargement
> LEFT ATRIUM: Moderate LA elongation

In MIMIC-IV, it appears that such information was flagged as PHI, and replaced with three underscores in the discharge reports. At least this is my understanding. What is now present in the notes is:
> LEFT ATRIUM: Mild ___
> LEFT ATRIUM: Moderate ___

Is this a known issue? Is there a possibility you may bring this information back?

**Comment by alistairewj** [comment]
We overhauled the deidentification process so there will be changes, mostly for the better. Raising issues like these are useful as our approach includes the ability to suppress false positives based on domain knowledge. So we could add a few context rules for these cases you've identified.

**Comment by burgersmoke** [comment]
Would it be helpful to report more instances which might be false positives?  I've noticed in some chief complaints, some crucial information is missing such as these:

`CC:  ___ pain`
`CC: ___, Wound Eval`
`CC: ___`

**Comment by alistairewj** [comment]
Yes, and including the `note_id` will make it easier to add these to the set of test cases we have. Thank you!
